src/inert.js

```diff
@@ -72,10 +73,10 @@ class InertRoot {
     // Watch for:
     // - any additions in the subtree: make them unfocusable too
     // - any removals from the subtree: remove them from this inert root's managed nodes
-    // - attribute changes: if `tabindex` is added, or removed from an intrinsically focusable element,
-    //   make that node a managed node.
+    // - attribute changes: if `tabindex` is added, or removed from an intrinsically focusable
+    //   element, make that node a managed node.
     this._observer = new MutationObserver(this._onMutation.bind(this));
-    this._observer.observe(this._rootElement, { attributes: true, childList: true, subtree: true });
```

Comment:
It's part of the Google ESLint rules:
https://github.com/google/eslint-config-google/blob/master/index.js#L246

"off" or 0 - turn the rule off
"warn" or 1 - turn the rule on as a warning (doesn’t affect exit code)
"error" or 2 - turn the rule on as an error (exit code is 1 when triggered)